### DISCUSSION TITLE
如何在envoy的采集指标envoy_cluster_upstream_rq增加url_path的label

### DISCUSSION THREAD

**Discussion by Guojie0922** [higress-group/higress#1471] [Q&A]
## Why do you need it?
 Is your feature request related to a problem? Please describe in details

## How could it be?
A clear and concise description of what you want to happen. You can explain more about input of the feature, and output of it.

**Comment by johnlanni** [comment]
我印象中 envoy 原生不支持 envoy_cluster_upstream_rq 上增加 route 信息，downstream 是可以的

**Reply by Guojie0922** [reply]
downstream应该怎么添加啊

**Comment by edenzhang2012** [comment]
我们也需要类似的功能，你们最后实现了吗？怎么实现的？能不能给个方案参考一下

---
We also need similar functions, have you finally implemented it? How is it achieved? Can you give me a plan for reference

**Comment by w7team** [comment]
同样关注这个问题

---
Also pay attention to this issue
